services/openshift-service-impl/src/test/java/org/kontinuity/catapult/service/openshift/impl/OpenShiftUrlTest.java

```diff
@@ -22,6 +23,7 @@
     private static final String DEFAULT_OPENSHIFT_URL = "https://localhost:8443";
 
     @Test
+    @Ignore("If setting the env var to run the test it will fail.")
```

Comment:
Oooh never @Ignore (for an acceptable definition of "never" ;) ), and def not as an unrelated change to some other issue.  Is this not passing for you?  If so, we need to open a bug and fix this immediately.  For the record it's passing for me locally and on the CI environment.  Could you explain what issues you're having?